### PR TITLE
Feat: update design for demo page lnb, gnb

### PR DESCRIPTION
**Pull request by youjung-hong**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

- 데모페이지: http://10.78.209.54:8080/examples/example01-basic.html

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/view/month/weekdayInMonthSchedule.hbs [hunk 1]
```diff
@@ -30,7 +30,7 @@
                     {{#if model.isFocused}}
                         background: #ffffff
                     {{else}}
-                        background:{{model.color}}
+                        background:{{model.borderColor}}
```
Comment (dongsik-yoo) [inline src/js/view/month/weekdayInMonthSchedule.hbs]: Why do you use `borderColor`? In `more.hbs` you used `bgColor` as calendar color.

File: examples/css/default.css [hunk 2]
```diff
@@ -1,236 +1,376 @@
+@import url(http://fonts.googleapis.com/earlyaccess/notosanskr.css);
+ 
+body {
+  font-family: 'Noto Sans KR', sans-serif;
```
[the remark this answers is not in the document]
Comment (youjung-hong) [reply examples/css/default.css]: Ok, I will change to Noto Sans

File: examples/css/default.css [hunk 3]
```diff
@@ -1,236 +1,376 @@
...
+  line-height: 30px;
 }
 
 .move-day {
-    border: 0;
+  padding: 8px;
+  font-size: 0;
+}
+
+#renderRange {
```
[the remark this answers is not in the document]
Comment (youjung-hong) [reply examples/css/default.css]: `render-range` is class name,
`renderRange` is id

File: examples/css/icons.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+input[type='checkbox'].checkbox-round + span {
+  display: inline-block;
+  cursor: pointer;
+  width: 14px;
+  height: 14px;
+  line-height: 14px;
+  vertical-align: middle;
+  margin-right: 8px;
+  border-radius: 8px;
+  border: solid 2px;
```
[the remark this answers is not in the document]
Comment (youjung-hong) [reply examples/css/icons.css]: the color `#333` comes from bootstrap.

File: examples/css/default.css [hunk 5]
```diff
@@ -1,236 +1,376 @@
...
-}
-
-.bi15 {
-    width: 15px;
-    height: 15px;
-}
-
 /** custom fontawesome */
+
 .fa {
```
[the remark this answers is not in the document]
Comment (youjung-hong) [reply examples/css/default.css]: #55 still have `fa` icons

File: src/js/view/month/weekdayInMonthSchedule.hbs (hunk 1 above)
Comment (youjung-hong) [reply src/js/view/month/weekdayInMonthSchedule.hbs]: @dongsik-yoo I changed `more.hbs` to use `borderColor`.
I think bullet color should be same to border color of all-day schedule

**Pull request merged**